kernel/opaqueproof.mli

```diff
@@ -46,12 +46,8 @@ module DiskData : sig
 end
 
 (* val mk_fetch : Constr.t Future.computation array Delayed.t -> opaque_disk_data *)
-val force_proof : DiskData.t -> opaquetab -> opaque -> constr
-val get_proof : DiskData.t -> opaquetab -> opaque -> constr Future.computation
-
-val force_constraints : opaquetab -> opaque -> Univ.ContextSet.t
-val get_constraints :
-  opaquetab -> opaque -> Univ.ContextSet.t Future.computation option
+val force_cst : DiskData.t -> opaquetab -> opaque -> constr * Univ.ContextSet.t
+val get_cst : DiskData.t -> opaquetab -> opaque -> (constr * Univ.ContextSet.t) Future.computation
```

Comment:
Why `_cst`? It is both the proof and the constraints, while cst seems to refer only to constraints